src/Workspaces/Core/Portable/PublicAPI.Unshipped.txt

```diff
@@ -2,6 +2,15 @@ Microsoft.CodeAnalysis.CodeStyle.NotificationOption.Severity.get -> Microsoft.Co
 Microsoft.CodeAnalysis.CodeStyle.NotificationOption.Severity.set -> void
 abstract Microsoft.CodeAnalysis.Editing.SyntaxGenerator.AliasImportDeclaration(string aliasIdentifierName, Microsoft.CodeAnalysis.SyntaxNode name) -> Microsoft.CodeAnalysis.SyntaxNode
 abstract Microsoft.CodeAnalysis.Editing.SyntaxGenerator.NameExpression(Microsoft.CodeAnalysis.INamespaceOrTypeSymbol namespaceOrTypeSymbol) -> Microsoft.CodeAnalysis.SyntaxNode
+const Microsoft.CodeAnalysis.Classification.ClassificationTypeNames.RegexAlternation = "regex - alternation" -> string
```

Comment:
This & licensing are my only concerns about going into `master`. @jinujoseph Who should we talk to about the public api change?

Reply:
/cc: @DustinCampbell @Pilchie

Reply:
Also @ManishJayaswal

Reply:
Assuming that these classifications would be returned by normal calls to the `Classifier.GetClassifiedSpansAsync(...)` outside of VS, these need to be public so that editors can handle them appropriately. I fully support adding them to the public API surface.

Reply:
Yaay.  Thanks @DustinCampbell